arangod/RocksDBEngine/RocksDBSettingsManager.cpp

```diff
@@ -154,7 +154,7 @@ Result RocksDBSettingsManager::sync(bool force) {
   auto minSeqNr = maxSeqNr;
 
   rocksdb::TransactionOptions opts;
-  opts.lock_timeout = 0.05; // do not wait for locking keys
+  opts.lock_timeout = 50; // do not wait for locking keys
```

Comment:
What does this do? I assume its unrelated?

Reply:
Yes, this is a random bugfix I made. `lock_timeout` is of type int64_t, not double.